lib/audited/audit.rb

```diff
@@ -105,6 +105,12 @@ def undo
       end
     end
 
+    def merge!(other)
+      raise ArgumentError, "#{other.inspect} should belong to the same auditable" \
+        if other.auditable_type != auditable_type || other.auditable_id != auditable_id
+      update_column(:audited_changes, other.audited_changes.merge(audited_changes))
```

Comment:
I think this method should be named `reverse_merge!`, so it is clear which one has precedence in a consistent manner to common ruby `merge!` methods, or named something completely different.

Reply:
Looks like it's clear that caller is taking precedence over `other` (like for common `merge` methods).
This line is same as
```
self.update_column(....
```
So we are updating self with contents of other.